docs/source/models/gbd2017/causes/tuberculosis/tuberculosis.rst

```diff
@@ -0,0 +1,164 @@
+.. _tuberculosis_model:
+
+=====================
+Modeling Tuberculosis
+=====================
+
+TB Overview
+-----------
+Tuberculosis (TB) is a disease caused by the bacteria Mycobacterium
+tuberculosis. It most commonly affects the lungs, although it can affect
+other parts of the body. TB can be fatal if not recognized and treated.
+It also can spread from person to person to infect others. Anyone who 
+inhales the airborne droplets that contains TB bacteria is called a "contact".
+A contact can be someone who living with the person who has infectious TB,
+like a family member, friend, or coworker. `UpToDate <https://www.uptodate.com/
+contents/tuberculosis-beyond-the-basics>`_
+
+What is Latent TB?
+------------------
+Latent tuberculosis infection (LTBI) is defined as a state of persistent
+immune response to stimulation by Mycobacterium tuberculosis antigens with
+no evidence of clinically manifest active TB. (WHO)
+
+What is Active TB?
+------------------
+Active tuberculosis, also known as reactivation TB. It occurs when individual's
+immune system becomes weakened and is no longer able to contain the latent bacteria;
+then the TB bacteria become "active", overwhelm the immune system and cause a person
+to become ill. (UpToDate) In GBD, Active TB consist of 6 causes: Drug-susceptible
+TB, Multidrug-resistant TB, Extensively drug-resistant TB, HIV/AIDS - Drug-susceptible
+TB, HIV/AIDS - Multidrug-resistant TB, and HIV/AIDS - Extensively drug-resistant TB.

```

Comment:
Can we say something about why GBD models TB and HIV together? Even just a single sentence explaining why this is important would be good.

Reply:
Also, I'm guessing that "extensively drug resistant" is worse than "multidrug resistant" -- is that correct? It might be nice to have a brief description of why there is a distinction between these two causes.